pkg/helper/versions/helpers.go

```diff
@@ -34,14 +34,12 @@ func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP
 		if isSTS && !ocm.HasSTSSupport(v.RawID(), v.ChannelGroup()) {
 			continue
 		}
-		if filterHostedCP {
-			valid, err := ocm.HasHostedCPSupport(v)
-			if err != nil {
-				return versionList, fmt.Errorf("failed to check HostedCP support: %v", err)
-			}
-			if !valid {
-				continue
-			}
+		valid, err := ocm.HasHostedCPSupport(v)
```

Comment:
I dont think this change is needed, we dont need to call HasHostedCPSupport if we dont need to filter on hosted CP